requirements/pip.txt

```diff
@@ -4,11 +4,11 @@
 #
 #    make upgrade
 #
-wheel==0.37.0
+wheel==0.37.1
     # via -r requirements/pip.in
 
 # The following packages are considered to be unsafe in a requirements file:
-pip==21.2.4
+pip==22.0.3
     # via -r requirements/pip.in
-setuptools==57.4.0
+setuptools==60.8.1
```

Comment:
We should add constraint to pip.in file as setuptools is being updated to version 60 here